cmake/testing.cmake

```diff
@@ -166,13 +166,19 @@ f3d_test(TestVolumeNonScalars waveletArrays.vti "300,300" "-vb --scalars=RandomP
 f3d_test(TestTextures WaterBottle.glb "300,300" "--geometry-only --texture-material=${CMAKE_SOURCE_DIR}/data/testing/red.jpg --roughness=1 --metallic=1 --texture-base-color=${CMAKE_SOURCE_DIR}/data/testing/albedo.png --texture-normal=${CMAKE_SOURCE_DIR}/data/testing/normal.png --texture-emissive=${CMAKE_SOURCE_DIR}/data/testing/red.jpg --emissive-factor=0.1,0.1,0.1")
 f3d_test(TestMetaDataImporter BoxAnimated.gltf "300,300" "-m")
 f3d_test(TestMultiblockMetaData mb.vtm "300,300" "-m")
-f3d_test_long_timeout(TestHDRI suzanne.ply "300,300" "--hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr")
-f3d_test_long_timeout(TestHDRIBlur suzanne.ply "300,300" "-u --hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr")
-f3d_test_long_timeout(TestHDRIEdges suzanne.ply "300,300" "-e --hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr")
 f3d_test(TestTIFF logo.tif "300,300" "-sy --up=-Y")
 f3d_test(TestUTF8 "(ノಠ益ಠ )ノ.vtp" "300,300")
 f3d_test(TestAnimationIndex InterpolationTest.glb "300,300" "--animation-index=7")
 
+f3d_test_long_timeout(TestHDRI suzanne.ply "300,300" "--hdri=${CMAKE_SOURCE_DIR}/data/testing/palermo_park_1k.hdr")
+set_tests_properties(TestHDRI PROPERTIES LABELS "HDRI")
```

Comment:
Can you just use a regexp based on the name in the CI?

Reply:
I could, it would exclude another HDRi related test which is not loading any hdri. Do you think that it would be better ?

Reply:
Ok. Just keep these changes then.